content/windows.md

```diff
@@ -563,11 +396,44 @@ Depending on how you installed Windows and which firmware your computer is runni
 - Press the `+` key (or `SHIFT` + `=`) to increase the number of seconds the system will wait before booting the default OS; press `-` to decrease the number of seconds.
 - Once a timer is set, the menu will display on every boot without having to hold down `SPACE`. (To reverse this, press `-` until the timer is back to zero.)
 
-## Repairing the Pop!_OS Bootloader
+##### Add Windows to systemd-boot
+
+If you installed Windows on a [dedicated drive](#installing-on-a-dedicated-drive), then the systemd-boot menu will not recognize the Windows drive right away. To make systemd-boot aware of the Windows installation, we will need to copy the Windows boot files from the Windows boot partition, to the Pop!\_OS boot partition.
+
+1. Boot to Pop!_OS
+2. Find the Windows drive
+    1. Run lsblk in a Terminal to list partitions on your systems
+        - The EFI partition should be the 1st partition on the Windows drive, and will be around 100M in size
+    2. Now mount the Windows EFI partition to /mnt
+        - sudo mount /dev/NAME where NAME is from the "name" column of the lsblk output for the EFI partition
+        - After mounting on a test machine, I ran ls -al /mnt/EFI and I was able to see a "Boot" and a "Microsoft" folder at this point
+    3. Now copy the "Microsoft" folder to your Pop!_OS EFI partition.
+        - will copy the folder if the Windows EFI was mounted as I indicated above
+        ```bash
+        sudo cp -Rv /mnt/EFI/Microsoft /boot/efi/EFI/Microsoft
+        ```
+3. When you reboot at this point, Windows should now show in the systemd-boot menu.
+
+#### EFI Boot Manager
+
+EFI Boot Manager or `efibootmgr` is a tool used to interact with an EFI system's boot entries. We can use `efibootmgr` in Linux to set Windows as our 1-time boot option. If we reboot the computer at this point, your computer will boot straight to Windows. Then, if you reboot the computer again, it will go back to Pop!_OS until you set the 1-time boot option again.
+
+Most system released after 2014 use EFI-based firmware, and can use this tool.
+
+1. Run the efibootmgr command in a Terminal.
+    - This will show you the available EFI boot entries found on your system
+2. Find the "Microsoft Boot Manager" line, we will need the 4-digit number found in the Boot000x section
```

Comment:
Conventionally, any number under 10 would be spelt out "four-digit, one-time", etc.

https://www.grammarly.com/blog/when-to-spell-out-numbers/

`Boot000x` should probably be in backtics.